PendingReleaseNotes

```diff
@@ -166,57 +166,59 @@
-of the feature, refer this link on how to perform it:
-https://docs.ceph.com/en/quincy/cephadm/upgrade/#staggered-upgrade
-Relevant tracker: https://tracker.ceph.com/issues/55715
+  ``<LEGACY_MAPPING_PREFIX><snapid>_<shardid>_<hobject_t::to_str()>`` to
+  ``<MAPPING_PREFIX><pool>_<snapid>_<shardid>_<hobject_t::to_str()>`` When this
+  change was introduced, 94ebe0e also introduced a conversion with a serious
+  bug that essentially destroyed legacy keys by mapping them to
+  ``<MAPPING_PREFIX><poolid>_<snapid>_`` without the object-unique suffix. The
+  conversion is fixed in this release. Relevant tracker:
+  https://tracker.ceph.com/issues/56147
+
+* The ``cephadm`` utility may now be configured to carry out CephFS MDS
+  upgrades without reducing ``max_mds`` to ``1``. Previously, Cephadm would
+  reduce ``max_mds`` to ``1`` in order to avoid certain potential
+  incompatibilities (for example: having two active MDS daemons modifying
+  on-disk structures with new versions, communicating
+  cross-version-incompatible messages). This action could be disruptive for
+  large-scale CephFS deployments because it is not easy for the cluster to
+  reduce active MDS daemons to ``1``.  NOTE: In order to take advantage of this
+  feature, staggered upgrade of the mons and mgrs might be necessary. For
+  details on how to perform staggered upgrades, see
+  https://docs.ceph.com/en/quincy/cephadm/upgrade/#staggered-upgrade Relevant
+  tracker: https://tracker.ceph.com/issues/55715
 
-* Introduced a new file system flag `refuse_client_session` that can be set using the
-`fs set` command. This flag allows blocking any incoming session
-request from client(s). This can be useful during some recovery situations
-where it's desirable to bring MDS up but have no client workload.
-Relevant tracker: https://tracker.ceph.com/issues/57090
+* A new filesystem flag ``refuse_client_session`` has been introduced. It is
+  set by running the ``fs set`` command. This flag allows incoming session
+  requests from clients to be blocked. In certain recovery situations, it can
+  be useful to bring up the Metadata Server but have no client workload.
+  Relevant tracker: https://tracker.ceph.com/issues/57090
 
-* New MDSMap field `max_xattr_size` which can be set using the `fs set` command.
-  This MDSMap field allows to configure the maximum size allowed for the full
-  key/value set for a filesystem extended attributes.  It effectively replaces
-  the old per-MDS `max_xattr_pairs_size` setting, which is now dropped.
-  Relevant tracker: https://tracker.ceph.com/issues/55725
+* A new MDSMap field ``max_xattr_size`` has been introduced. It is set by 
+  running the ``fs set`` command. This MDSMap field is used to specify the
+  maximum size allowed for the full key-value set of a filesystem's extended
+  attributes. It replaces the existing per-MDS ``max_xattr_pairs_size``
+  setting, which has been eliminated.  Relevant tracker:
```

Comment:
@zdover23 - Ditto. Also, the pattern for the rest of the document is that _Relevant tracker: [link]_ begins on a new line. But I have no idea whether that's something worth preserving.